config/schema/base_document_type.json

```diff
@@ -10,13 +10,16 @@
     "latest_change_note",
     "link",
     "mainstream_browse_pages",
+    "mainstream_browse_page_content_ids",
     "organisations",
+    "organisation_content_ids",
     "policies",
     "popularity",
     "public_timestamp",
     "publishing_app",
     "rendering_app",
     "specialist_sectors",
+    "topics",
```

Comment:
From a chat IRL with Tijmen, we decided to change this to `topic_content_ids` (which is clearer as this is a list of `content_ids`) and on expansion (another PR), add an extra field called `topics` with the expanded topics.